crmsh/bootstrap_lock.py

```diff
@@ -26,37 +26,24 @@ class JoinLock(object):
     SSH_TIMEOUT = 10
     SSH_OPTION = "-o ConnectTimeout={} -o StrictHostKeyChecking=no".format(SSH_TIMEOUT)
     SSH_EXIT_ERR = 255
-    MIN_JOIN_TIMEOUT = 120
-    WAIT_INTERVAL = 10
 
-    def __init__(self, init_node):
+    def __init__(self, init_node=None):
         """
         Init function
         """
         self.init_node = init_node
+        self.is_joining = True if self.init_node else False
```

Comment:
I wouldn't call this `joining`. I would call it `remote`. Basically we are implementing a remote lock now

Reply:
Changed